targets/PythonSdk/source/setup.py

```diff
@@ -0,0 +1,20 @@
+import setuptools
+with open("README.md", "r") as fh:
+        long_description = fh.read()
+        
+setuptools.setup(
+    name="playfab",
+    version="0.0.1",
+    author="Todd Bello",
+    author_email="toddbell@microsoft.com",
+    description="PlayFab SDK for Python",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/PlayFab/PythonSdk",
+    packages=setuptools.find_packages(),
+    classifiers=(
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: Apache Software License",
+        "Operating System :: OS Independent",
```

Comment:
Look in the unreal playfab.uplugin for a list of typical marketplace/package tags.